packetbeat/_meta/fields.yml

```diff
@@ -1898,6 +1898,14 @@
               description: >
                 The algorithm used for the certificate's signature.
 
+            - name: alternative_names
+              type: array
+              description: Subject Alternative Names for this certificate.
+
+            - name: raw
+              type: keyword
```

Comment:
I think our index templates use an ignore_above value of 1024 bytes. Will the PEM values hit that limit?

https://www.elastic.co/guide/en/elasticsearch/reference/current/ignore-above.html
https://github.com/elastic/beats/blob/master/libbeat/template/processor.go#L143

Reply:
I'd say that's ok, they won't be indexed but are still returned in `source`. We could consider making them `text`, but not sure how "searchable" the content is anyway.